Procurement/ViewModel/Filters/ForumExport/FullBestiaryOrbFilter.cs

```diff
@@ -16,7 +16,7 @@ public string Keyword
         {
             get
             {
-                return "Full Bestiary Orb";
+                return "Captured Beasts";
```

Comment:
"Captured" beasts include those not just stored in orbs.  I used the name "Full Bestiary Orb" because that's essentially what GGG calls them -- see the file name used for the [icon](https://web.poecdn.com/image/Art/2DItems/Currency/BestiaryOrbFull.png): `BestiaryOrbFull.png`.

Reply:
I used captured beasts because it's the same term used on trade, the other sounds more like an internal name